packages/venia-concept/src/components/SignIn/signIn.js

```diff
@@ -0,0 +1,93 @@
+import { createElement, Component } from 'react';
+import PropTypes from 'prop-types';
+import Input from 'src/components/Input';
+import Button from 'src/components/Button';
+import defaultClasses from './signIn.css';
+import classify from 'src/classify';
+import Form from 'src/components/Form';
+import ErrorDisplay from 'src/components/ErrorDisplay';
+
+class SignIn extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            signInSection: PropTypes.string,
+            signInDivider: PropTypes.string,
+            forgotPassword: PropTypes.string,
+            root: PropTypes.string,
+            signInError: PropTypes.string,
+            showCreateAccountButton: PropTypes.string
+        }),
+
+        signInError: PropTypes.object,
+        signIn: PropTypes.func
+    };
+
+    state = {
+        password: '',
+        username: ''
+    };
+
+    get errorMessage() {
+        const { signInError } = this.props;
+        return <ErrorDisplay error={signInError} />;
+    }
+
+    render() {
+        const { classes, showCreateAccountForm } = this.props;
+        const { onSignIn, errorMessage } = this;
+
+        return (
+            <div className={classes.root}>
+                <Form submitForm={onSignIn}>
+                    <Input
+                        onChange={this.updateUsername}
+                        helpText={'example help text'}
+                        label={'Username or Email'}
+                        required={true}
+                    />
+
+                    <Input
```

Comment:
When I load this field in my browser, Chrome gives me an accessibility warning:

![image](https://user-images.githubusercontent.com/1643758/46089511-4b63d100-c174-11e8-849d-ab1dcd62ce73.png)

And links me here: https://www.chromium.org/developers/design-documents/create-amazing-password-forms

It looks like the autocomplete attribute would be easy to add here, and would help mobile devices know to activate their password managers.

Reply:
Nice catch - I had verbose warnings off in Chrome so I missed this.